recipe/meta.yaml

```diff
@@ -30,7 +30,7 @@ requirements:
     - jedi >=0.10
     - pexpect  # [unix]
     - pickleshare
-    - prompt_toolkit >=2.0.0,<4,!=3.0.0,!=3.0.1
+    - prompt-toolkit >=2.0.0,<4,!=3.0.0,!=3.0.1
```

Comment:
@beckermr b/c we had both versions in the past conda can get confused. Recently I updated a few feedstocks to "link" the `-`_ and `_` via multiple outputs. This will allow us to use a canonical package name in the build while users can specify either package name or import name at install time.